CMGTools/TTHAnalysis/python/tools/angular_vars.py

```diff
@@ -0,0 +1,121 @@
+from CMGTools.TTHAnalysis.treeReAnalyzer import *
+
+class angular_vars:
+    def __init__(self):
+ 	self.branches = [ "L_phi", 
+			  "B_loose_phi", "B_medium_phi", "B_tight_phi", 
+			  "DR_l_b_loose1", "DR_l_b_medium1", "DR_l_b_tight1", 
+			  "DR_l_b_loose2", "DR_l_b_medium2", "DR_l_b_tight2",
+			  "pTl_DR_l_b_loose1", "pTl_DR_l_b_medium1", "pTl_DR_l_b_tight1", 
+			  "pTl_DR_l_b_loose2", "pTl_DR_l_b_medium2", "pTl_DR_l_b_tight2",  
+			  "L_pt" ]
+    def listBranches(self):
+        return self.branches[:]	
+    def __call__(self,event):
+        # make python lists as Collection does not support indexing in slices
+	leps = [l for l in Collection(event,"LepGood","nLepGood",4)]
+        jets = [j for j in Collection(event,"Jet","nJet25",8)]
+	bloose  = [j for j in jets if j.btagCSV > 0.423]
```

Comment:
@milosdjordjevic, could you please check these working points and update them to the latest recommended ones if needed?